lib/crypto/c_src/digest.c

```diff
@@ -114,6 +114,23 @@ static struct digest_type_t digest_types[] =
 #endif
     },
 
+    {"shake128", "SHAKE-128", 0, 0,
+#ifdef HAVE_SHAKE128
+    {&EVP_shake128, NULL}
+#else
+    {NULL, NULL}
+#endif
+    },
+
+    {"shake256", "SHAKE-256", 0, 0,
+#ifdef HAVE_SHAKE256
+    {&EVP_shake256, NULL}
+#else
+    {NULL}
```

Comment:
this line is superfluous – left from a change while adapting to changes in digest.c
```suggestion
```